projects/react/src/lib/useMaskito.ts

```diff
@@ -0,0 +1,57 @@
+import {Maskito, MASKITO_DEFAULT_OPTIONS, MaskitoOptions} from '@maskito/core';
+import {RefCallback, useCallback, useEffect, useLayoutEffect, useState} from 'react';
+
+const useIsomorphicLayoutEffect =
+    typeof window !== 'undefined' ? useLayoutEffect : useEffect;
+
+export type MaskitoPredicate = (
+    element: HTMLElement,
+) => HTMLInputElement | HTMLTextAreaElement;
+
+const defaultMaskitoPredicate: MaskitoPredicate = e =>
+    e.querySelector('input,textarea') || (e as HTMLInputElement | HTMLTextAreaElement);
+
+/**
+ * Hook for convenient use of Maskito in React
+ * @description For controlled inputs use `onInput` event
+ * @param options options used for creating Maskito
+ * @param predicate function that can help find nested Input or TextArea
+ * @returns ref callback to pass it in React Element
+ * @example
+ * // To avoid unnecessary hook runs with Maskito recreation pass named variables
+ * // good example ✅
+ * useMaskito({ options: maskitoOptions, predicate: maskitoPredicate })
+ *
+ * // bad example ❌
+ * useMaskito({ options: { mask: /^.*$/ }, predicate: () => e.querySelector('input') })
+ */
+export const useMaskito = ({
+    options = MASKITO_DEFAULT_OPTIONS,
+    predicate = defaultMaskitoPredicate,
```

Comment:
Let's call it 
```suggestion
    maskitoElement = defaultMaskitoPredicate,
```

Just to be similar to the angular API
https://github.com/Tinkoff/maskito/blob/bbcfd084370d1517f12ab3e9e0761b24f7a3094e/projects/angular/src/lib/maskito.directive.ts#L22

Reply:
`maskitoElement` sounds more like kind of an instance than a function

and imo no need to make identical variable names

Reply:
`predicate` is not enough meaningful too.

Let' call it, at least, `elementPredicate` / `queryPredicate`

Reply:
done